docs/intro/install.rst

```diff
@@ -1,245 +1,56 @@
-or in user-space. We do not recommend installing Scrapy system wide.
-
-Instead, we recommend that you install Scrapy within a so-called
-"virtual environment" (`virtualenv`_).
-Virtualenvs allow you to not conflict with already-installed Python
-system packages (which could break some of your system tools and scripts),
-and still install packages normally with ``pip`` (without ``sudo`` and the likes).
+.. _Python Package Index: https://pypi.org/
 
-To get started with virtual environments, see `virtualenv installation instructions`_.
-To install it globally (having it globally installed actually helps here),
-it should be a matter of running::
-
-    $ [sudo] pip install virtualenv
-
-Check this `user guide`_ on how to create your virtualenv.
-
-.. note::
-    If you use Linux or OS X, `virtualenvwrapper`_ is a handy tool to create virtualenvs.
-
-Once you have created a virtualenv, you can install Scrapy inside it with ``pip``,
-just like any other Python package.
-(See :ref:`platform-specific guides <intro-install-platform-notes>`
-below for non-Python dependencies that you may need to install beforehand).
-
-Python virtualenvs can be created to use Python 2 by default, or Python 3 by default. As Scrapy
-only supports Python 3, make sure you created a Python 3 virtualenv.
-
-.. _virtualenv: https://virtualenv.pypa.io
-.. _virtualenv installation instructions: https://virtualenv.pypa.io/en/stable/installation/
-.. _virtualenvwrapper: https://virtualenvwrapper.readthedocs.io/en/latest/install.html
-.. _user guide: https://virtualenv.pypa.io/en/stable/userguide/
-
-
-.. _intro-install-platform-notes:
-
-Platform specific installation notes
-====================================
 
+.. _install-conda:
 .. _intro-install-windows:
 
-Windows
--------
+Installing from conda-forge
+===========================
 
-Though it's possible to install Scrapy on Windows using pip, we recommend you
-to install `Anaconda`_ or `Miniconda`_ and use the package from the
-`conda-forge`_ channel, which will avoid most installation issues.
-
-Once you've installed `Anaconda`_ or `Miniconda`_, install Scrapy with::
+If you are using Anaconda_ or Miniconda_, you can install Scrapy from the
+`conda-forge`_ channel::
 
   conda install -c conda-forge scrapy
 
-
-.. _intro-install-ubuntu:
-
-Ubuntu 14.04 or above
----------------------
-
-Scrapy is currently tested with recent-enough versions of lxml,
-twisted and pyOpenSSL, and is compatible with recent Ubuntu distributions.
-But it should support older versions of Ubuntu too, like Ubuntu 14.04,
-albeit with potential issues with TLS connections.
-
-**Don't** use the ``python-scrapy`` package provided by Ubuntu, they are
-typically too old and slow to catch up with latest Scrapy.
-
-
-To install Scrapy on Ubuntu (or Ubuntu-based) systems, you need to install
-these dependencies::
-
-    sudo apt-get install python3 python3-dev python3-pip libxml2-dev libxslt1-dev zlib1g-dev libffi-dev libssl-dev
```

Comment:
In past I found it helpful to have a list of packages needed to get `pip install scrapy` working. It can be too much to try to find this information in install docs of lxml, cryptography, searching on stackoverflow why packages which require a compiler don't work (python3-dev), etc. This list can become outdated, but in practice it doesn't happen that often.